src/voters/services/voters.service.ts

```diff
@@ -280,6 +285,34 @@ export class VotersService {
     return columns
   }
 
+  async querySearchColumn(searchColumn: string, electionState: string) {
+    let searchValues: string[] = []
+    try {
+      const searchUrl = `${API_BASE}/customer/application/column/values/1OSR/VM_${electionState}/${searchColumn}?id=1OSR&apikey=${L2_DATA_KEY}`
+      type ExpectedResponse = { values: string[]; message: string }
```

Comment:
Do we want to move this inline type out to a types file like we're doing elsewhere? 🤔